docs/xml/enumerations/vote_variation.rst

```diff
@@ -1,35 +1,43 @@
 VoteVariation
 =============
 
-.. todo::
-
-   Document VoteVariation
 
 +----------------------+----------------------------------------------------------------------------------+
 | Name                 | Description                                                                      |
 |                      |                                                                                  |
 +======================+==================================================================================+
-| 1-of-m               |                                                                                  |
-+----------------------+----------------------------------------------------------------------------------+
-| approval             |                                                                                  |
+| 1-of-m               | A method where each voter can select up to one candidate.                        |
 +----------------------+----------------------------------------------------------------------------------+
-| cumulative           |                                                                                  |
+| approval             | `Approval voting`_, where each voter can select as many candidates as desired.   |
 +----------------------+----------------------------------------------------------------------------------+
-| majority             |                                                                                  |
+| cumulative           | `Cumulative voting`_, where each voter can distribute their vote to up to        |
+|                      | *N* candidates.                                                                  |
 +----------------------+----------------------------------------------------------------------------------+
-| n-of-m               |                                                                                  |
+| majority             | A 1-of-m method where the winner needs more than 50% of the vote to be elected.  |
 +----------------------+----------------------------------------------------------------------------------+
-| plurality            |                                                                                  |
+| n-of-m               | A method where each voter can select up to *N* candidates.                       |
 +----------------------+----------------------------------------------------------------------------------+
-| proportional         |                                                                                  |
+| plurality            | A 1-of-m method where the candidate with the most votes is elected, regardless   |
```

Comment:
There are no non-1-of-m elections (e.g., approval, cumulative) which can be won via a plurality?